.github/workflows/integration-actions.yml

```diff
@@ -0,0 +1,248 @@
+        working-directory: ./src/app
+
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Update Docker Compose
+        run: cp test/conf/docker-compose.yml docker/docker-compose/docker-compose.yml
+        working-directory: ./
+
+      - name: Run Docker Beer Garden
+        run: docker-compose up -d beer-garden
+        working-directory: ./docker/docker-compose
+
+      - name: Wait for Beer Garden to intialize
+        run: sleep 45
+
+      - name: Grab logs from Beer-Garden
+        run: docker-compose logs --tail 100 beer-garden
+        working-directory: ./docker/docker-compose
+
+      - name: Uninstall Brewtils
+        run: pip${{ matrix.python-version }} uninstall brewtils
+
+      - name: Install Test Version of Brewtils
+        run: pip${{ matrix.python-version }} install brewtils==${{matrix.plugin-brewtils}}
+
+      - name: Install Python Deps
+        run: pip${{ matrix.python-version }} install -r requirements.txt
+        working-directory: ./test/integration
+
+      - name: Copy Helper Classes
+        run: cp -r helper remote_plugins/
+        working-directory: ./test/integration
+
+      - name: Test Plugins
+        run: python${{ matrix.python-version }} -m pytest remote_plugins/
+        working-directory: ./test/integration
+
+      - name: Shutdown Docker Containers
+        run: docker-compose stop
+        working-directory: ./docker/docker-compose
+
+  Local-Plugin-Testing:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        python-version: ['3.7']
+        os: ['ubuntu-18.04']
+      fail-fast: false
+
+    name: Local Plugins - Python ${{ matrix.python-version }}
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.4
+
+      - name: Build Local Beer Garden Docker
+        run: make docker-build-unstable
+        working-directory: ./src/app
+
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Update Docker Compose
+        run: cp test/conf/docker-compose.yml docker/docker-compose/docker-compose.yml
+        working-directory: ./
+
+      - name: Checkout Local Plugins
+        uses: actions/checkout@v2
+        with:
+          repository: beer-garden/example-plugins
+          path: ./tmp
+
+      - name: Move Test Plugins
+        run: |
+          cp -r ./tmp/echo ./docker/docker-compose/data/localplugins
+          cp -r ./tmp/complex ./docker/docker-compose/data/localplugins
+          cp -r ./tmp/dynamic ./docker/docker-compose/data/localplugins
+          cp -r ./tmp/sleeper ./docker/docker-compose/data/localplugins
+          cp -r ./tmp/echo-sleeper ./docker/docker-compose/data/localplugins
+          cp -r ./tmp/error ./docker/docker-compose/data/localplugins
```

Comment:
Can we just check these out directly to the tmp directory? With this if we add a new plugin we'll have to update this list too.

Reply:
I first started off with checking out the plugins directly. It technically works, but I didn't like how Child plugin was throwing errors when the downstream Beer Garden connected. I guess that is a fine issue to to have. Also, for the example plugins it works best if we have integration tests for them. So originally I only added those.